Spec-readme.md

```diff
@@ -0,0 +1,38 @@
+## Spec Command
+
+```
+stencil spec [filePath]
+stencil sp [filePath]
+```
+
+Example: `stencil spec spec.yaml` or `stencil spec src/spec.yaml`
+
+**Description**
+
+Bootstraps the project based on given specification file.
+
+**Arguments**
+
+| Argument  |  Description |
+|-----------|--------------|
+|  `[filePath]`	 | The specification file along with path |
+
+## Structure of spec.yaml
+```
+stencil: 0.0.1
+
+info:
+  properties:
+    project-name: "ProjectName"
+    package-manager: "npm" 
+    
+tooling: [logging]
+
+endpoints:

```

Comment:
What is going to be the content of this key? Can't see that detailed out anywhere.

Reply:
Once the crud api get's merge I'll update the endpoints structure.
Also wanted to know how should it work?
Can you give some suggestions here?